src/bin/bench-tps.rs

```diff
@@ -391,6 +391,17 @@ fn compute_and_report_stats(
     );
 }
 
+// First transfer 3/4 of the tokens to the dest accounts
+// then ping-pong 1/4 of the tokens back to the other account
+// this leaves 1/4 token buffer in each account
+pub fn should_switch_directions(num_tokens_per_account: i64, i: i64) -> bool {
+    if i % (num_tokens_per_account / 4) == 0 && (i >= (3 * num_tokens_per_account) / 4) {
```

Comment:
This function doesn't need to be `pub`, doesn't need that if-then-else, and probably doesn't need most those parens. Great tests though, want to try this:

```rust
fn should_switch_directions(num_tokens_per_account: i64, i: i64) -> bool {
    i % (num_tokens_per_account / 4) == 0 && i >= 3 * num_tokens_per_account / 4
}
```